app/resources/views/voting-lists/show.blade.php

```diff
@@ -49,7 +49,7 @@
                             }}
                         @endif
 
-                        @if ($votingList->vote->primaryVote)
+                        @if ($votingList->vote->primaryVote?->votingList)
```

Comment:
Vielleicht sollten wir Accessors für die URL für `Vote` und `VotingList` einführen, die ggf. `null` returnen. Wäre dann glaube ich an der Stelle etwas einfacher zu lesen:

```suggestion
                        @if ($votingList->vote->primaryVote->url)
```